conan/internal/api/audit/providers.py

```diff
@@ -191,10 +191,19 @@ def _get(self, ref):
                     "query": textwrap.dedent(full_query)
                 }
             )
+
+            if response.status_code == 404:
+                ConanOutput().write("\n")
+                ConanOutput().warning(
+                    f"An error occurred while connecting to the '{self.name}' provider.\n"
+                    "This is likely because your JFrog Platform instance does not have JFrog Curation.\n"
+                    "For more information, visit: https://audit.conan.io/missing-curation\n"
```

Comment:
URL to discuss

Reply:
We can have an entry in the Conan FAQ in the docs, same as we have for "Missing binaries" error message.